src/pages/Portfolio/ImpactDashboard/DistributionGraphs.vue

```diff
@@ -23,8 +23,6 @@ import { gql } from 'graphql-tag';
 import delayUntilVisibleMixin from '#src/plugins/delay-until-visible-mixin';
 import getCacheKey from '#src/util/getCacheKey';
 import StatsTable from '#src/components/Stats/StatsTable';
-import KvPieChart from '@kiva/kv-components/vue/KvPieChart';
-import KvTreeMapChart from '@kiva/kv-components/vue/KvTreeMapChart';
```

Comment:
Don't we still need treemap here as it's used by default below? Or maybe the "chart" prop isn't used any longer...

Reply:
That prop actually isn't used. I removed the computed value when I removed these imports, but looks like I missed the unused prop.